CMakeLists.txt

```diff
@@ -97,9 +97,6 @@ message(STATUS "-------------------------------------------\n")
 set(GZ_PHYSICS_ENGINE_RELATIVE_INSTALL_DIR
   ${GZ_LIB_INSTALL_DIR}/gz-${GZ_DESIGNATION}-${PROJECT_VERSION_MAJOR}/engine-plugins
 )
-set(GZ_PHYSICS_ENGINE_INSTALL_DIR
```

Comment:
I think this cmake variable is still being used in different engine plugins' CMakeLists.txt for windows install. Not sure if we can remove this.

Reply:
Thanks. I see. Considering that this is the only change, should we close this PR and leave the macro as it is?

Reply:
Leave it as deprecated, then. [97f5255](https://github.com/gazebosim/gz-physics/pull/705/commits/97f52559839befde01e5ac4ded1aae1477f24fbf). Thanks!